### ISSUE TITLE
Migrate to the jest-community org?

### ISSUE THREAD

**Issue by orta** [kalcifer/awesome-jest#2]
Hey there @pksjce @rahulcs @koresar 

We're starting an official awesome-jest repo and wanted to have these contributions to it, given that this repo is a great starting point for us to build on: https://github.com/jest-community/awesome-jest

Any chance you want to contribute some of your work over to here ^?

**Comment by koresar** [comment]
Hey, it's hard to follow the project structure. Could you please insert this text to the appropriate position?

```
## Videos
- [Fun introduction to Jest (30 min)](https://www.youtube.com/watch?v=tvy0bSgwtTo)
```

That's the line I contributed to this project. :)

**Comment by pksjce** [comment]
@orta - Sure, will add my links to the resources part of your repo.
